src/promptflow/promptflow/_sdk/_service/templates/ui_traces.j2

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Table</title>
+    <style>
+        table, th, td {
+            border: 1px solid black;
+            border-collapse: collapse;
+        }
+        th, td {
+            padding: 5px;
+            text-align: left;
+        }
+    </style>
+</head>
+<body>
+    <table>
+        <thead>
+            <tr>
+                <th>trace_id</th>
+                <th>span_id</th>
+                <th>content</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for row in data %}
+            <tr>
+                <td>{{ row.trace_id }}</td>
```

Comment:
we should dynamic show the inter fields

Reply:
not sure if it's easy to do; I understand this is a dummy table, so we don't need to really make this dynamic?